src/stories/3.Tx.stories.jsx

```diff
@@ -28,6 +30,37 @@ const dummyTx = {
   network: 'main'
 }
 
+const dummyAccountTokens = [
+  {
+    address: '0xD87C62BfA2A4DFAbdD1Fad5A1cb73a63345aA2B0',
+    name: 'Stellar',
+    symbol: 'XLM',
+    balance: '20',
+    decimals: '18'
+  },
+  {
+    address: '0x123456BfA2A4DFAbdD1Fad5A1cb73a63345aA2B0',
+    name: 'Golem',
+    symbol: 'GO',
+    balance: '33',
+    decimals: '10'
+  }
+]
+
+const dummyWallet = {
+  name: 'ETHER',
+  symbol: 'ETHER',
+  address: '0x123456BfA2A4DFAbdD1Fad5A1cb73a63345zzzzz',
+  balance: '15'
+}
+
+storiesOf('Tx/Currency Select', module)
+  .add('default ', () => <CurrencySelect />)
```

Comment:
We use 'default' a lot in the stories but I've been trying to stay away from that since it doesn't really mean anything.

In this case, I think these stories would be great:
* No ether, no tokens
* Ether, no tokens
* No ether, multiple tokens

And any other edge cases you can think of